config.c

```diff
@@ -250,8 +268,25 @@ static int config_sysctl_apply(char *key, char *value, void *opaque)
 	return 0;
 }
 
-static int pv_config_load_config_from_file(char *path,
-					   struct pantavisor_config *config)
+static int pv_config_load_policy(const char *policy,
+				 struct dl_list *config_list)
+{
+	char path[PATH_MAX];
+
+	if (!policy)
+		return 0;
+
+	pv_paths_etc_policy_file(path, PATH_MAX, policy);
+	printf("DEBUG: loading policy from %s\n", path);
```

Comment:
do we really want to print that always regardless of log level? can we not use pv_log here?

Reply:
if I change it to pv_log it doesn't work for some reason. It's supposed to print to stdout if logserver is not initialized, but it doesn't